### ISSUE TITLE
Handle file, line and column attributes of messages

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-webui#100]
I guess ideally we'd like messages with location info to have links that bring you to the correct location in the input file (and messages with unknown location would be ignored). A quicker solution could be to show the info when hovering the message, or just append it to the messages if we are not concerned about the readability of the log. File info should be displayed as relative paths.

**Comment by josteinaj** [comment]
@bertfrees should we keep this issue open? If so, I guess this would require changes to the engine, so maybe it then should be moved to the daisy/pipeline issue tracker?

**Comment by bertfrees** [comment]
Yes you are right, this is not really an issue until it is properly supported in the engine. I'd move the issue if I could.

**Comment by bertfrees** [comment]
@rdeltour It is possible to move a Github issue to another repository, but only admins can do that.
